antlr/ContactQL.g4

```diff
@@ -1,34 +1,51 @@
 grammar ContactQL;
 
-// rebuild with % antlr4 -Dlanguage=Go ContactQL.g4 -o ../contactql/gen -package gen -visitor
+// rebuild with % antlr -Dlanguage=Go ContactQL.g4 -o ../contactql/gen -package gen -visitor
 
 import LexUnicode;
 
-fragment HAS : [Hh][Aa][Ss];
-fragment IS  : [Ii][Ss];
-
-LPAREN     : '(';
-RPAREN     : ')';
-AND        : [Aa][Nn][Dd];
-OR         : [Oo][Rr];
-COMPARATOR : ('=' | '!=' | '~' | '>=' | '<=' | '>' | '<' | HAS | IS);
-TEXT       : (UnicodeLetter | UnicodeDigit | '_' | '.' | '-' | '+' | '/' | '@')+;
-STRING     : '"' (~["] | '""')* '"';
-
-WS         : [ \t\n\r]+ -> skip;        // ignore whitespace
-
-ERROR      : . ;
-
-parse      : expression EOF;
-
-expression : expression AND expression  # combinationAnd
-           | expression expression      # combinationImpicitAnd
-           | expression OR expression   # combinationOr
-           | LPAREN expression RPAREN   # expressionGrouping
-           | TEXT COMPARATOR literal    # condition
-           | TEXT                       # implicitCondition
-           ;
-
-literal : TEXT                          # textLiteral
-        | STRING                        # stringLiteral
-        ;
\ No newline at end of file
+fragment HAS: [Hh][Aa][Ss];
+fragment IS: [Ii][Ss];
+
+LPAREN: '(';
+RPAREN: ')';
+AND: [Aa][Nn][Dd];
+OR: [Oo][Rr];
+COMPARATOR: (
+		'='
+		| '!='
+		| '~'
+		| '>='
+		| '<='
+		| '>'
+		| '<'
+		| HAS
+		| IS
+	);
+TEXT: (
+		UnicodeLetter
+		| UnicodeDigit
+		| '_'
+		| '.'
+		| '-'
+		| '+'
+		| '/'
+		| '@'
+	)+;
+STRING: '"' (~["] | '\\"')* '"';
```

Comment:
This line is the only thing that changed in this file besides vscode insisting it be re-formatted.

Changing contact QL to use same (golang) string escaping as Excellent2 instead of the old Excel/Excellent1 `""` syntax.. otherwise things get really weird.

Took a peek in the TextIt database and I can't see any queries involving string escaping.